src/test/scala/chiselTests/CompatibilityInteroperabilitySpec.scala

```diff
@@ -17,9 +18,8 @@ object CompatibilityComponents {
     val a = UInt(width = 32)
     val b = UInt(width = 32).flip
   }
-  class ChiselRecord extends Record {
+  class ChiselRecord extends Record with AutoCloneType {
```

Comment:
```suggestion
  class ChiselRecord extends Record {
```
The idea is to make `AutoCloneType` happen automatically, NOT to require everyone to mix it in.